walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -218,46 +139,47 @@ In this task, you use the Identity Management service to protect both the client
 
 === Add Identity Management service to the client app (PWA) and create a realm for users
 
-. Navigate to the link:{mdc-url}[Mobile Developer Console, window="_blank"].
-. Click on the *showcase-app* card to display the configuration screen.
-. Click the *Mobile Services* tab.
-. Click *Bind to App* for *Identity Management* in the *Unbound Services* section and then click the *next* button.
-.. When prompted for an *Admin User Name* enter:
-+
-----
-admin
-----
-.. When prompted for an *Admin User Password* enter:
-+
-----
-admin
-----
-. Complete the wizard.
-This process also creates a new realm in the {customer-sso-name}.
+. As your admin to create new realm for your user and obtain url to be able to login to the keycloak instance
+. In new realm we going to create two new clients - one for server side and one for the client side application.
+. Open URL obtained from adminstrator
+. Login to keycloak (you will be asked to provide additional details like email and name)
+. When completing login process you should see the main keycloak realm page
+. In the right menu choose *Clients* under *Configure* in the menu.
+.. Click on the *Create* button on the top-right corner to start creating a new client.
+.. Enter `showcase-app-client` for Client ID.  Do not change the rest of the fields.
+.. Save the client. You should see the *Settings* page of the client.
+.. You should see the *Settings* page of the client.
+.. Change the `Valid Redirect URIs` to '{route-ionic-showcase-server-host}*'
+.. Change the `Web Origins` to '*'
+.. Click on the *Installation* tab, and select `Keycloak OIDC JSON` format. Use the `Download` button to save the configuration file to a directory on the computer.
 
-. Navigate to the *Configuration* tab of the *mobile app* in the Mobile Developer Console.
+. Navigate to the project config-maps section {openshift-host}/console/project/{walkthrough-namespace}/browse/config-maps
+. Select `webapp-config`
+. Add new `auth` section in config by utilizing the config that was copied in previous step.
+. In configuration replace `auth-server-url` to `url` and `resource` to `clientId`
```

Comment:
This can be done in showcase but it is more valuable to show users that this needs to be done for their apps rather than hiding this fact in the showcase implementation.

Reply:
is there any particular reason that we ask developer to modify the downloaded config for the showcase app?

Reply:
it is not for showcase app itself. It is that SSO adapter for js require it (even in their docs).

Reply:
https://www.keycloak.org/docs/latest/securing_apps/index.html#_javascript_adapter :)

The format that SSO provides is old because I think we use some old version there.